task_templates/pipelines/python3_sklearn_multiclass/custom.py

```diff
@@ -75,88 +70,50 @@ def fit(
         raise Exception("Running multiclass estimator task: class_order expected to be not None")
     estimator.fit(X, y)
 
-    # You must serialize out your model to the output_dir given, however if you wish to change this
-    # code, you will probably have to add a load_model method to read the serialized model back in
-    # When prediction is done.
-    # Check out this doc for more information on serialization https://github.com/datarobot/custom-\
-    # model-templates/tree/master/custom_model_runner#python
-    # NOTE: We currently set a 10GB limit to the size of the serialized model
+    # Dump the trained object [in this example - a trained Logistic Regression pipeline]
+    # into an artifact [in this example - artifact.pkl]
+    # and save it into output_dir so that it can be used later when scoring data
+    # Note: DRUM will automatically load the model when it is in the default format (see docs)
+    # and there is only one artifact file
     with open("{}/artifact.pkl".format(output_dir), "wb") as fp:
         pickle.dump(estimator, fp)
+
+    # Save class labels for future reference if needed
```

Comment:
I don't think we should say "if needed", because it's very much required to define a score hook...

Reply:
@MikhailYakubovskiy why is this required? We can use the class labels from kwargs, right? Also the pytorch_multiclass template doesn't save class labels...

Reply:
I'm actually now thinking more about it. Can kwargs contain a list of classes sorted differently than what sklearn estimator uses? if yes, then in the score() we should probably use model.classes_ instead of kwargs. Thoughts?

Reply:
Good call! I checked and it is a different order. For sklearn only I switched it to use model.classes_ for the column headers in the score hook

Reply:
I guess for non-sklearn they can also somehow pass the sequence through the artifact. E.g. they can pickle a dictionary where they keep both a trained model and a list of classes in the right sequence...

Reply:
So I was trying to think through that. Looking at our internal predict function that's called if the client doesn't provide theirs, we just use the kwargs that are passed through. So I assume that's correct?

For sklearn we use model.classes_